R/transform_point_model_out.R

```diff
@@ -0,0 +1,75 @@
+#' Transform mean/median model output into a forecast object
+#'
+#' @param model_out_tbl Forecast model output tibble
+#' @param target_data Observed 'ground truth' data to be compared against forecasts
+#' @param output_type Forecast output type: "mean" or "median"
+#'
+#' @return forecast_point
```

Comment:
I note that documentation pages were created for these functions that are designed to be internal functions on the netlify preview of the documentation site. The [tidyverse style guidelines](https://style.tidyverse.org/documentation.html#internal-functions) suggest using the `@noRd` tag so the markdown documentation pages are not created for those functions.

Reply:
A note that initially @kimberlynroosa had added the `@noRd` tag but that was causing some very weird warnings in R CMD CHECK which we could not get to the bottom of. Removing the `@noRd` tag silenced the cryptic warning. I have a quiet hope that when there is an actual function with a `man` page, the `@noRd` tag can be applied to these internal functions with no issue...but that's just a hunch/hope!